src/wazuh_modules/wm_control.c

```diff
@@ -58,28 +58,60 @@ char* getPrimaryIP(){
         }
     }
 
+    OSHash *gateways = OSHash_Create();
+    if (getGatewayList(gateways) < 0){
+        merror("Error creating the list of gateways");
```

Comment:
` mterror(WM_CONTROL_LOGTAG, "...)`. Review the messages as well: `Unable to obtain the Default Gateway list`